### DISCUSSION TITLE
Utter confusion

### DISCUSSION THREAD

**Discussion by birdie-github** [JamesCJ60/Universal-x86-Tuning-Utility#97] [General]
I'm completely lost in the utility:

1. What's the currently selected preset? There's seemingly no indication anywhere.
2. How can I edit/configure the premade presets?
3. What are the defaults (max CPU temperature and power consumption) for my APU?
4. How can I reset the APU to built-in defaults? Reboot?
5. Notifications, when selecting presets, are shown twice.
6. "Magpie FSR" doesn't work. Nothing happens when I press it.
7. "System Info" also doesn't work. Likewise.
8. There's a weird white line at the top of the application.
9. Premade presets seem to be optimized for a single 30W APU model. AMD APUs range from 15 to over 100W AFAIK.

Using 2.0.0 beta build 9.1 under Windows 10 LTSC 2021.

<img width="810" alt="screenshot2" src="https://github.com/JamesCJ60/Universal-x86-Tuning-Utility/assets/4958603/40076f95-de90-4d2a-b2dc-0d04dccbe42b">

**Comment by Charliecoop** [comment]
I'll do my best to say what I've worked out:
What's the currently selected preset? There's seemingly no indication anywhere.
- No way to tell. I select 'Balanced' and then apply a custom preset which *I think* overwrites the changed settings if you do it in that order. It's confusing.

How can I edit/configure the premade presets?
- You can't.

What are the defaults (max CPU temperature and power consumption) for my APU?
- You can't find this out unless your laptop brand tells you (likely they won't). Usually it's 95C for CPU temp but it's safe to move it to 105 if you're on AMD Ryzen I think.

How can I reset the APU to built-in defaults? Reboot?
- Only an uninstall does this as far as I figured.

Notifications, when selecting presets, are shown twice.
- I didn't notice this.

"Magpie FSR" doesn't work. Nothing happens when I press it.
- The dev has said they removed FSR for the moment. No reason given.

"System Info" also doesn't work. Likewise.
- Likely removed without a reason given (there is one, just we don't know).

There's a weird white line at the top of the application.
- It's a bug.

Premade presets seem to be optimized for a single 30W APU model. AMD APUs range from 15 to over 100W AFAIK.
- I think this is focused more for laptops.

Sorry this isn't great answers, but this is what I think - feel free to correct me, maybe we'll work it out. Keep in mind the developer managed to do something that is wildly popular and isn't getting paid. I think this project needs more people to help them fix bugs and get things more mainstream as it's very barebones but still such an impressive bit of software in terms of what it can do for laptops that are locked.

Devs if you're reading this. Good luck, we know you do this for free essentially, we're impressed with what you've achieved already!
